templates/home.phtml

```diff
@@ -141,8 +141,8 @@
             <button id="nav-mark"><?= \F3::get('lang_markread')?></button>
 
             <div id="nav-filter-wrapper">
-            <h2 id="nav-filter-title" class="nav-filter-expanded" tabindex="0"><span role="link"  aria-expanded="true"><?= \F3::get('lang_filter') ?></span></h2>
-            <ul id="nav-filter">
+            <span role="heading" aria-level="2"><h2 id="nav-filter-title" class="nav-filter-expanded" tabindex="0" role="link" aria-expanded="true"><?= \F3::get('lang_filter') ?></h2></span>
+            <ul id="nav-filter" aria-label="<?= \F3::get('lang_filters_list_label') ?>">
```

Comment:
> Would it make sense to use `aria-labelledby`?

aria-label is fine here since for aria-labeledby we are required to show/define the corresponding string somewhere on the page and then we will have to retrieve the ID of it.

Reply:
Also why did you create new strings? Or would it make sense to make them more verbose for the screen reader? Like “Filter by source”

Reply:
> Also why did you create new strings? Or would it make sense to make them more verbose for the screen reader? Like “Filter by source”

Actually I had to create the new strings in order to asign the labels for the lists.
And this makes retrieving the lists very pleasant by the screen reader users because just by the label only they can recognise that what the list is for.
Implementing the aria-label for the lists is very good and leaves a good impression on the user about their navigation experience.